backend/app/main/dto/relativeDto.py

```diff
@@ -8,6 +8,6 @@ class RelativeDto:
         'abstract': fields.String(required=True, description='relative title'),
         'year': fields.Integer(required=True, description='relative year'),
         'cluster': fields.Integer(required=True, description='relative keyword cluster'),
-        'citations': fields.List(fields.String),
+        'references': fields.List(fields.String),
```

Comment:
thinking about this line: maybe a `migrate` should have been enough for me.

Reply:
Yep, definitely, rescraping could have been an issue as I changed these _citation/reference_ thingies.